assets/css/style.css

```diff
@@ -1173,55 +1183,84 @@ pre code::-webkit-scrollbar-thumb {
     color: var(--white-color);
 }
 
-.cardLink:hover { text-decoration: none; }
+.cardLink:hover {
+    text-decoration: none;
+}
 
 
 /* Table of Contents */
 
-#toc{
-    background-color: #f7f7f7;
+#toc {
+    background-color: var(--white-color);
     border: 1px solid #6D6E71;
     box-sizing: border-box;
     border-radius: 4px;
     padding-inline-start: 2rem;
     padding-inline-end: 2rem;
 }
 
-#toctitle{
+#toctitle {
     font-weight: 600;
     font-size: 1.5rem;
     color: #1CABE2;
     margin: 1.5rem 0rem;
 }
 
 .content > .toc > ul li::before {
-    display:none;
+    display: none;
 }
 
-.content > .toc > .sectlevel1 > li a{
+.content > .toc > .sectlevel1 > li a {
     font-weight: 400;
     font-size: 1.2rem;
     color: #4C4C4C;
     margin: 0.5rem 0rem;
 }
 
-.content > .toc > .sectlevel1 > li ul li a{
-    color:#677993;
-    font-size:1rem;
+.content > .toc > .sectlevel1 > li ul li a {
+    color: #677993;
+    font-size: 1rem;
 }
 
-.sectlevel2{
-    padding-top:1rem;
+.sectlevel2 {
+    padding-top: 1rem;
 }
 
-@media only screen and (max-width: 768px){
+@media only screen and (max-width: 768px) {
     .sectlevel2 {
-        display:none;
+        display: none;
     }
-    
+
     .content > .toc > .sectlevel1 > li a:hover, a:focus + .sectlevel2 {
-        display:block;
-        padding-top:1rem;
+        display: block;
+        padding-top: 1rem;
     }
 
 }
+
+/* Changes to the footer and header links */
+
+.change-color-on-hover:hover {
+    text-decoration: underline !important;
+}
+
+.footer-social-icons:hover {
+    color: var(--body-color);
+    background-color: var(--white-color);
+    border-radius: 25%;
+}
+
+/* Just a try */
+.navbar-toggler:focus,
+.navbar-toggler:active {
+    outline: 0;
+}
+
+/* hide close when burger shown */
+.navbar-toggler.collapsed .close-icon>b {
+    display: none !important;
+}
+
+.navbar-toggler:not(.collapsed) .navbar-toggler-icon {
+    display: inline;
```

Comment:
Curious again, what do you intend to try out in these lines?

Reply:
Actually those lines are to hide the close in case the hamburger is shown so as to avoid both appearing at once.